modelgen/src/test/java/cz/cvut/kbss/jopa/modelgen/OutputFilesGeneratorTests.java

```diff
@@ -122,48 +128,29 @@ void isAnnotatedWithFailTest() {
     @Nested
     class GenerateOutputFiles {
 
-        @AfterEach
-        void deleteSMClass() {
-            File file = new File("./src/test/java/cz/test/ex/TestingClass_.java");
-            file.delete();
-        }
-
         @Test
         void containsIdentifier() {
-            assertTrue(actualResult.contains("public static volatile Identifier<TestingClass, URI> uri;"));
+            assertTrue(actualResult.contains("public static volatile Identifier<TestingClassOWL, Test> uri;"));
         }
 
         @Test
         void containsSingular() {
-            assertTrue(actualResult.contains("public static volatile SingularAttribute<TestingClass, TestingClass> testingClass;"));
+            assertTrue(actualResult.contains("public static volatile SingularAttribute<TestingClassOWL, Test> object;"));
         }
 
         @Test
         void containsTypedProperties() {
-            assertTrue(actualResult.contains("public static volatile PropertiesSpecification<TestingClass, Map, URI, Object> propertie;"));
+            assertTrue(actualResult.contains("public static volatile PropertiesSpecification<TestingClassOWL, Test> properties;"));
         }
 
         @Test
         void containsListAttribute() {
-            assertTrue(actualResult.contains("public static volatile ListAttribute<TestingClass, String> listAttribute;"));
+            assertTrue(actualResult.contains("public static volatile ListAttribute<TestingClassOWL, Test> data-list;"));
         }
 
         @Test
         void containsSetAttribute() {
-            assertTrue(actualResult.contains("public static volatile SetAttribute<TestingClass, String> setAttribute;"));
-        }
-
-
-        String readFileAsString(File file) {
-            try {
-                Scanner scanner = new Scanner(file);
-                String fileContents = scanner.useDelimiter("\\Z").next();
-                scanner.close();
-                return fileContents;
-            } catch (IOException e) {
-                System.err.println("Chyba cteni");
```

Comment:
This catch is useless (in the static method on line 159). Throw a runtime exception wrapping the IO exception or declare the whole method to throw the IOException.